notebooks/data-sources/TestGrid/metrics/test_pass_failures.ipynb

```diff
@@ -20,12 +20,12 @@
   },
```

Comment:
<p>Since this dataset is not yet added to the repository, do we want to continue using "testgrid_810" or add this one to the repo? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>this updated dataset is being added to the repo as a part of this PR</p>

Reply:
Perfect :+1: